src/common/components/clusterConfiguration/SecurityFields.tsx

```diff
@@ -3,17 +3,15 @@ import { useFormikContext } from 'formik';
 import { Checkbox, FormGroup } from '@patternfly/react-core';
 
 import { RenderIf } from '../ui/RenderIf';
-import { ExternalLink, getFieldId, HelperText, TextAreaField, trimSshPublicKey } from '../ui';
+import { getFieldId, HelperText, TextAreaField, trimSshPublicKey } from '../ui';
 import { Cluster } from '../../api';
 import { NetworkConfigurationValues } from '../../types/clusters';
-import { SSH_GENERATION_DOC_LINK } from '../../config';
 
 export const SshPublicKeyHelperText: React.FC<{
   fieldId?: string;
 }> = ({ fieldId = 'sshPublicKey' }) => (
   <HelperText fieldId={fieldId}>
-    Paste the content of a public ssh key you want to use to connect to the hosts into this field.{' '}
-    <ExternalLink href={SSH_GENERATION_DOC_LINK}>Learn more</ExternalLink>
+    Generate a key with <i>ssh-keygen -o</i> and add the value of <i>~/.ssh/id_rsa.pub</i> here.{' '}
```

Comment:
This looks like it's probably old in the design. The previous code looks nicer, plus it links to the documentation that explains how to set it. @nirfarkas could you confirm?

Current display (it's in a few other places too)
![ssh-key](https://user-images.githubusercontent.com/829045/178296653-b7f9deca-2077-4a5a-945f-915d4bc7b654.png)

Reply:
This is from the task, but maybe it's not updated: 
![image](https://user-images.githubusercontent.com/69599321/178435217-90f2255d-712f-442f-8ffa-020a895f4853.png)